### PR TITLE
Barebones set of what's available

### PR DESCRIPTION
**Pull request by mbleigh**
This is just a quick scaffold to get things started and to have a meta package of the new lines available for some of the other work that's going on.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "dependencies": {
+    "iron-ajax":        "PolymerElements/iron-ajax#master",
+    "iron-selector":    "PolymerElements/iron-selector#master",
+    "iron-icon":        "PolymerElements/iron-icon#master",
+    "iron-icons":       "PolymerElements/iron-icon#master",
+    "iron-iconset":     "PolymerElements/iron-iconset#master",
+    "iron-iconset-svg": "PolymerElements/iron-iconset-svg#master",
+    "iron-image":       "PolymerElements/iron-image#master",
+    "iron-doc-viewer":  "PolymerElements/iron-doc-viewer#master",
+    "iron-meta":        "PolymerElements/iron-meta#master"
```
Comment (cdata) [inline bower.json]: We are no longer using branches to refer to dependencies. Please change the dependency version to `#^0.8.0` for all of these.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "iron-elements",
+  "version": "0.0.0",
```
Comment (cdata) [inline bower.json]: Let's use 0.8.0.

File: .bowerrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "directory": ".."
```
Comment (cdata) [inline .bowerrc]: We no longer check in this Bower configuration, as it is very dangerous for unwary element developers. Please remove the checked-in `.bowerrc`.

**Comment by mbleigh** [comment]
@cdata done and done.

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "iron-icons":       "PolymerElements/iron-icon#^0.8.0",
+    "iron-iconset":     "PolymerElements/iron-iconset#^0.8.0",
+    "iron-iconset-svg": "PolymerElements/iron-iconset-svg#^0.8.0",
+    "iron-image":       "PolymerElements/iron-image#^0.8.0",
+    "iron-doc-viewer":  "PolymerElements/iron-doc-viewer#^0.8.0",
+    "iron-meta":        "PolymerElements/iron-meta#^0.8.0"
+  },
+
+  "homepage": "https://github.com/PolymerElements/iron-elements",
+  "authors": ["Google Polymer Project Authors"],
```
Comment (notwaldorf) [inline bower.json]: nit: I think we've been using "The Polymer Authors" as authors lately

**Comment by notwaldorf** [comment]
Would you mind squashing the commits, please? :)

**Comment by mbleigh** [comment]
Squashed.

**Comment by cdata** [comment]
Looking good, thanks for the contribution!

**Comment by notwaldorf** [comment]
Would you mind holding off merging this for a bit? I don't think the CLA bot is running on this org yet.

**Comment by notwaldorf** [comment]
Nevermind, @azakus says we're good CLA wise.

**Pull request merged**